Start to build logging

pritchardians opened the pull request:
Start to build logging

jasonwpacker reviewed (commented):
Can you look into this linting error?


Run # stop the build if there are Python syntax errors or undefined names
./src/utils/__init__.py:1:6: E999 SyntaxError: invalid syntax
from    ^
1     E999 SyntaxError: invalid syntax
1
Error: Process completed with exit code 1.

pritchardians commented:
Ah yes. I added code to the __init__ file without even noticing. Great test case for our checks, though!
Fixed it. I will push a commit and request a new pull request
